src/routes/+page.svelte

```diff
@@ -62,8 +63,9 @@
 </section>
 
 <!-- GTM Problem -->
-<section class="flex flex-col gap-8 bg-neutral-800 p-8">
-	<Text type="title1" color="darkPrimary" class="text-center">The GTM Problem</Text>
+<section class="flex flex-col items-center gap-8 bg-neutral-800 p-8">
+    <span class="p-2 rounded-md bg-opo-orange"><Users class="size-6 md:size-8 fill-primary-dark" /></span>
```

Comment:
Since we're repeating this pattern of wrapping the icon in a `<span>`, with the same classes being used, quite a lot, I'm wondering if we should just create an `Icon` component co-located to this page `src/routes/Icon.svelte` and have this in that component 🤔

Reply:
Awesome! I've created a separate component just to keep the icon style consistent with this commit – https://github.com/OnePlusOneInc/oneplusoneinc.github.io/pull/4/commits/ed459c645c76f9103864c1a61cf7363353bb2e1a ✅